modules/dxp/apps/multi-factor-authentication/app.bnd

```diff
@@ -1,15 +1,15 @@
 Liferay-Releng-App-Description:
 Liferay-Releng-App-Title: ${liferay.releng.app.title.prefix} Multi-Factor Authentication
 Liferay-Releng-Bundle: true
-Liferay-Releng-Category: Security
+Liferay-Releng-Category: foundation
```

Comment:
@csierra sorry I should have reviewed this more closely yesterday... "Security" was correct. Can you please send one more follow up? You can send it directly to BChan and `ci:reopen`, since it's a trivial change.

The directions on the app.bnd Grow article are misleading, I will fix the article.

@ling-alan-huang can you please fix this on SF (including `format-source-current-branch`) as well? We should only allow these categories:
```
Communication
Productivity
Security
Utility
```

Reply:
@jpince, I got it.